### PR TITLE
updating caption

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @cody-carlson
- [ ] @cody-carlson-learning
- [ ] @GregoireWulliamoz-eaton
- [ ] @MarleneHE
- [ ] @VarunEaton
- [ ] @breakpoint7
- [ ] @gds1968
- [ ] @git-september
- [ ] @jimlane
- [ ] @mmg99

### PR DESCRIPTION
**Pull request by gds1968**
it updates comments
this PR closes  #2 
##Done
-commit
- branch creation

### CODE REVIEW

**Comment by gds1968** [comment]
good

File: _slides/05-gds1968.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Blabla
```
Comment (cody-carlson-learning) [inline _slides/05-gds1968.md:6]: LGTM! 🥳

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# GitHub for Developers
+# GitHub for Developers - GDS
```
Comment (cody-carlson-learning) [inline README.md]: looks good over here, also

**Review by cody-carlson-learning — APPROVED** [review]
approved for changes, looks great!

File: README.md (hunk 2 above)
Comment (cody-carlson-learning) [inline README.md]: hey maybe we add an exclamation point

**Review by cody-carlson-learning — CHANGES_REQUESTED** [review]
grammar

File: README.md (hunk 2 above)
Comment (cody-carlson-learning) [reply README.md]: ok let's add it next time

**Review by cody-carlson-learning — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (gds1968) [inline README.md]: comment too short !

**Review by gds1968 — COMMENTED** [review]
#That's ##cool

File: README.md (hunk 2 above)
Comment (cody-carlson-learning) [inline README.md]: exclamation point maybe?

**Review by cody-carlson-learning — CHANGES_REQUESTED** [review]
hey how about this instead?

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# GitHub for Developers
+# GitHub for Developers - GDS!
```
Comment (cody-carlson-learning) [inline README.md:1]: test

**Review by cody-carlson-learning — CHANGES_REQUESTED** [review]
test

**Review by cody-carlson-learning — APPROVED** [review]

**Pull request merged**